### PR TITLE
Metastatic status

### PR DESCRIPTION
**Pull request by aricohen93**
## Description
- Added new patterns for metastasis detection. Developed on CT-Scan reports.
- Added citation of articles

## Checklist

- [x] If this PR is a bug fix, the bug is documented in the test suite.
- [x] Changes were documented in the changelog (pending section).
- [x] If necessary, changes were made to the documentation (eg new pipeline).

### CODE REVIEW

**Review by Thomzoy — APPROVED** [review]
All good, thanks !

File: tests/pipelines/ner/disorders/test_all.py [hunk 1]
```diff
@@ -91,7 +94,7 @@ def check(self):
             doc = self.nlp(text)
             ents = doc.spans[self.disorder]
 
-            assert len(ents) == int(has_match)
+            assert len(ents) >= int(has_match)
```
Comment (percevalw) [inline tests/pipelines/ner/disorders/test_all.py:97]: @Thomzoy @aricohen93 should we keep this change ?

Comment (percevalw) [inline tests/pipelines/ner/disorders/test_all.py:97]: @Thomzoy @aricohen93 should we keep this change ?

**Review by percevalw — APPROVED** [review]
Thanks !

**Pull request merged**